engines/utils/PatrowlEngine.py

```diff
@@ -191,12 +191,12 @@ def stop_scan(self, scan_id):
         res.update({"status": "success"})
         return jsonify(res)
 
-
+    # Stop all scans
     def stop(self):
-        res = {"page": "stop all scans"}
+        res = {"page": "stop"}
```

Comment:
stopscans or stop ?

Reply:
I suggest "stopscans". All engines should be reviewed to ensure consistency

Reply:
I'll check all the engines tonight if I can